randovania/data/json_data/prime1/Phendrana Drifts.txt

```diff
@@ -59,6 +59,10 @@ Asset id: 4146616697
   * Normal Door to Ruins Entryway/Door to Phendrana Shorelines
   > Door to Shoreline Entrance
       Trivial
+  > Door to Temple Entryway
+      All of the following:
+          Combat/Scan Dash (Intermediate)
+          Scan Visor or X-Ray Visor
```

Comment:
Cant this be done in any visor? If so, could be an invisible objects intermediate or something added on to this

Reply:
The only viable scan point is the flicker bats. They cannot be locked onto with combat visor, but they can be locked onto with x-ray.